app/models/miq_expression/tag.rb

```diff
@@ -0,0 +1,20 @@
+class MiqExpression::Tag
+  def self.parse(tag)
+    klass, ns = tag.split(".")
+    ns = "/" + ns.split("-").join("/")
+    ns = ns.sub(/(\/user_tag\/)/, "/user/") # replace with correct namespace for user tags
+    new(klass.constantize, ns)
+  end
+
+  attr_reader :model, :namespace
+
+  def initialize(model, namespace)
+    @model = model
+    @namespace = namespace
+  end
+
+  def contains(value)
```

Comment:
if this is meant to return a boolean, perhaps `contains?` is a better method name?